src/crimson/osd/objclass.cc

```diff
@@ -356,7 +356,11 @@ int cls_cxx_map_clear(cls_method_context_t hctx)
 
 int cls_cxx_map_write_header(cls_method_context_t hctx, bufferlist *inbl)
 {
-  return 0;
+  OSDOp op{ CEPH_OSD_OP_OMAPSETHEADER };
+  op.indata.claim(*inbl);
+  auto ret = execute_osd_op(hctx, op);
```

Comment:
```suggestion
  return execute_osd_op(hctx, op);
```
`ret` looks unnecessary. It doesn't bring any extra information about the programmer's intention, so let's just `return` instead.